system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -217,6 +248,83 @@
                 :VariableLevel1 "EMISSIONS"}]
               (:ScienceKeywords (:umm concept)))))))
 
+(deftest bulk-update-remove-all-platforms-test
```

Comment:
I think this test should make sure that what you remove later is present first. This will be redundant with other tests, for sure, but I think it's a good practice and will give more certainty to the test/setup/etc.